gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/api/ApiListItem.java

```diff
@@ -54,14 +54,10 @@
     @Schema(description = "The last date (as a timestamp) when the API was updated.", example = "1581256457163")
     private Date updatedAt;
 
-    @Schema(description = "The visibility of the API regarding the portal.", example = "PUBLIC", allowableValues = { "PUBLIC", "PRIVATE" })
+    @Schema(description = "The visibility of the API regarding the portal.", example = "PUBLIC")
     private Visibility visibility;
 
-    @Schema(
-        description = "The status of the API regarding the gateway.",
-        example = "STARTED",
-        allowableValues = { "INITIALIZED", "STOPPED", "STARTED", "CLOSED" }
-    )
+    @Schema(description = "The status of the API regarding the gateway.", example = "STARTED")
```

Comment:
The `Lifecycle.State` enum is in another library and I don't think we will add OAI annotation in it.
I think we should keep these values.
wdyt ?
https://github.com/gravitee-io/gravitee-common/blob/master/src/main/java/io/gravitee/common/component/Lifecycle.java#L23-L29

Reply:
[See comment above](https://github.com/gravitee-io/gravitee-api-management/pull/1318#discussion_r814039826)